cms/tests/test_extensions.py

```diff
@@ -31,8 +31,9 @@ def test_register_extension(self):
         page_extension = self.get_page_extension_class()
 
         # register first time
-        extension_pool.register(page_extension)
+        register_extension_result = extension_pool.register(page_extension)
```

Comment:
Maybe a better test is just to use the decorator to register one of the test extensions and check if it's registered in the test.
This looks a bit "hackish"